src/providers/deno.rs

```diff
@@ -9,6 +9,18 @@ use crate::nixpacks::{
 };
 use anyhow::{Context, Result};
 use regex::Regex;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Default, Debug)]
+
+pub struct DenoTasks {
+    pub start: Option<String>,
```

Comment:
Not sure about this. I believe deno tasks can have other tasks aswell? (ie, db migrations)

Reply:
Yea that's fine. Serde should just drop it on the floor. Can add that stuff later if we wanna action on it

Reply:
We should handle deno tasks like we handle package.json scripts IMO.